### PR TITLE
Feature/33 modify script colors

### LINKED ISSUE #33: Modify colors scripts
# REQUIREMENTS
Modify the scripts that hold the colors used in the app and set the final ones.

# DESIGN
Just the same colors at the wireframe design: 
https://www.figma.com/file/IGRyLwNFX0GKXkikAfzfKw/Tribo?node-id=18%3A2

# Acceptance Criteria
List all required acceptance criteria
- [ ] The colors must match the ones in the style guide
- [ ] The colors must be accessible from every part of the app

### PR DESCRIPTION
**Pull request by alcamk**
# Description
This feature is related to #33 
 This file only contains the naming convention and the hexadecimal text for the colors 
 Examples:
N/A
 
 # Testing
N/A 
  
  ## Screenshots 
N/A

### CODE REVIEW

File: tribomobile/__tests__/App-test.js [hunk 1]
```diff
@@ -1,14 +1,27 @@
 /**
  * @format
  */
```
Comment (fernandoamz) [inline tribomobile/__tests__/App-test.js]:
```suggestion

```

File: tribomobile/__tests__/App-test.js [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
 import App from '../App';
+import colors from '../src/colors';
 
-// Note: test renderer must be required after react-native.
-import renderer from 'react-test-renderer';
+const FixedDimensionsBasics = () => {
+  return (
+    <View>
+      <View
+        style={{width: 50, height: 50, backgroundColor: colors.GreenContent}}
```
Comment (fernandoamz) [inline tribomobile/__tests__/App-test.js]: The inline styles are not recommended. Could you move the lines where you use inline styles to the stylesheets object?

File: tribomobile/components/modals/LittlePinInfo.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    alignItems: 'center',
+  },
+  descriptionLeft: {
+    maxWidth: '75%',
+    paddingHorizontal: 8,
+  },
+  descriptionTitle: {
+    fontSize: 15,
+    fontWeight: 'bold',
+    color: '#828282',
```
Comment (fernandoamz) [inline tribomobile/components/modals/LittlePinInfo.js]: Standardize the colors on the themes file into shared components and call it here.

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by Ing-Heber — DISMISSED** [review]

**Review by develiasdaniel — DISMISSED** [review]
review the comments of mentor
Good job

**Review by Ocosmee — DISMISSED** [review]

**Review by tovl1 — DISMISSED** [review]
Nice

**Comment by Ing-Heber** [comment]
Well done Oscar!

File: tribomobile/screens/registrationScreen.js [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+              />
+              <RegistrationCheckbox
+                checkboxLabel={contentText.textRegistrationScreenCheckboxSabado}
+              />
+              <RegistrationCheckbox
+                checkboxLabel={
+                  contentText.textRegistrationScreenCheckboxDomingo
+                }
+              />
+            </View>
```
Comment (LeyaimMagma) [inline tribomobile/screens/registrationScreen.js]: Can you refactor this?

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]
This PR has conflicts

**Review by LeyaimMagma — APPROVED** [review]

**Review by JordiGN — APPROVED** [review]

**Review by tovl1 — APPROVED** [review]

**Review by Ocosmee — APPROVED** [review]

**Pull request merged**